project/plugins.sbt

```diff
@@ -11,6 +11,7 @@ addSbtPlugin("com.github.sbt" % "sbt-native-packager" % "1.9.11")
 addSbtPlugin("io.spray" % "sbt-revolver" % "0.9.1")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.4.3")
 addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.10.1")
-addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.4.7")
+addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.4.8")
+ThisBuild / libraryDependencySchemes += "org.scala-native" % "sbt-scala-native" % "always"
```

Comment:
Honestly, It's not wise to have this enabled. Either `sbt-scala-native-crossproject` should release a new version with fresh Scala Native.
```
[error] java.lang.RuntimeException: found version conflict(s) in library dependencies; some are suspected to be binary incompatible:
[error] 
[error] 	* org.scala-native:sbt-scala-native:0.4.8 (early-semver) is selected over 0.3.7
[error] 	    +- default:http4s-build:0.1.0-SNAPSHOT (sbtVersion=1.0, scalaVersion=2.12) (depends on 0.4.8)
[error] 	    +- org.portable-scala:sbt-scala-native-crossproject:1.2.0 (sbtVersion=1.0, scalaVersion=2.12) (depends on 0.3.7)
```

Reply:
So, I dropped a request for the new release of the plugin https://github.com/portable-scala/sbt-crossproject/issues/149.

Reply:
D'oh! As it always happens, @armanbilge already has done an investigation of this a few days ago. 
TLDR. Do we simply need to wait for the `0.4.9` release of Scala Native to resolve this?

Reply:
Yes, my plan was to wait and see what happens. As with Scala.js, I don't think there is an urgency to get on the latest Scala Native release.

In this case, effectively every project would need this workaround 😅 and I can't even put it in sbt-typelevel, because it's not a problem in the build but a problem in the meta-build.

Sorry I didn't mention anything before!